src/main/java/co/zecko/retailer/client/httpClient/CollectionClient.java

```diff
@@ -14,11 +14,15 @@
 import org.apache.commons.lang3.StringUtils;
 
 @FieldDefaults(level = AccessLevel.PRIVATE)
-public class ZeckoCollectionClient extends BaseClient {
+public class CollectionClient extends BaseClient {
 
     static final String BASE_URI = "/retailer/v1/collections";
+    public String zeckoAccessToken;
```

Comment:
```suggestion
    final String zeckoAccessToken;
```

Reply:
@akartik80 we can't make it final coz we are changing it on line 28:
`zeckoAccessToken = getZeckoAccessToken(zeckoAccessToken);`